cypress/e2e/movie_details_spec.cy.js

```diff
@@ -0,0 +1,25 @@
+describe('movie details spec', () => {
+  beforeEach(() => {
+    cy.intercept("GET", "http://localhost:3001/api/v1/movies/389"), {
+      statusCode: 200
+    }
+    cy.visit('http://localhost:3000/')
+  })
```

Comment:
I think this probably isn't working because you need to reference the fixture file after the status code. There is also an extra closing parenthesis after the url on line 3. 

I think we'll also need to add an intercept for the home page too, since you start on the home page to get to the movie details page.